src/components/User/Avatar.js

```diff
@@ -19,6 +20,14 @@ const Avatar = ({ href, src, className }) => {
         </a>
       </AvatarContainer>
     )
+  } else if (username) {
+    container = (
+      <AvatarContainer className={className}>
+        <Link to={`/user/${username}`}>
+          <img src={src} alt="avatar" />
```

Comment:
Possibly a bit too big. Maybe width 150px or something like that?

<img width="603" alt="Screenshot 2019-07-05 at 14 47 03" src="https://user-images.githubusercontent.com/2630/60726595-d8259180-9f33-11e9-827a-f57faefaf392.png">